### ISSUE TITLE
Post-metadata is in the bottom of the post instead of below the title

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#4729]
The post&#45;metadata with information about the author, the data etc is places in the bottom of the post. On the designs, this information is placed just below the header.
![In production](https://github.com/ushahidi/platform/assets/8624777/72c4f062-60ec-4af1-935b-a45a6e8a9fff)
![From designs](https://github.com/ushahidi/platform/assets/8624777/e6df12f4-3e3f-46c9-a0a9-4368bce04162)

---
Created automatically from USH-1090 (https://linear.app/ushahidi/issue/USH-1090/post-metadata-is-in-the-bottom-of-the-post-instead-of-below-the-title)

**Comment by mannyAndem** [comment]
Hi, can I get started working on this?

**Comment by mannyAndem** [comment]
Fixed the issue and created a [pr](https://github.com/ushahidi/platform-client-mzima/pull/805) on the platform-client-mzima :)

**Comment by mannyAndem** [comment]
@Angamanga :)

Already got my environment all set up.

![Image](https://github.com/ushahidi/platform/assets/127485113/18bbca9f-603b-4fa0-a6ca-583ca7bb6d7f)

**Comment by Angamanga** [comment]
@mannyAndem Thank you! I will review shortly!

**Comment by mannyAndem** [comment]
Thank you!
Can I get started on another issue or does this need to be merged first?

**Comment by Angamanga** [comment]
@mannyAndem You can get started on another issue in the meantime :) Just ping me on one that is not assigned yet.

**Comment by Angamanga** [comment]
@mannyAndem How are you getting on with this issue? Need any guidance?

**Comment by mannyAndem** [comment]
Already fixed the issue and submitted a [pr](https://github.com/ushahidi/platform-client-mzima/pull/805) which I had said earlier. Must have slipped your mind @Angamanga :)

**Comment by Angamanga** [comment]
Thank you for reminding me @mannyAndem :)
